src/internal/operators/audit.ts

```diff
@@ -56,6 +56,7 @@ export function audit<T>(durationSelector: (value: T) => SubscribableOrPromise<a
     let hasValue = false;
     let lastValue: T | null = null;
     let durationSubscriber: Subscriber<any> | null = null;
+    let isComplete = false;
 
     const endDuration = () => {
```

Comment:
@cartant, do we want to fix the duration notifier treating completion like a notification here? Or shall we get that at a later time? The whole thing is weird. Like `[]` "Here's your value, in this array!". My gut tells me it should be a `noop`, and I think we all agreed on that.

Reply:
Yeah, we should fix it. Whether it's fixed here or someplace else, I don't mind. I intend to do another pass through all of the operators that take a notifier to make sure their behaviour is consistent, etc.